src/styles/main.scss

```diff
@@ -1,3 +1,224 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+:root {
+  --background-color: #fff;
+  --main-accent-color: #060b35;
+  --blue-hover-color: #34568b;
+  --second-color: #616070;
+  --blue-accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0,0.1);
+  width: 100%;
+}
+
+.header__logo {
+  display: block;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
   margin: 0;
 }
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+  font-weight: 500;
+  font-size: 12px;
+
+  color: var(--main-accent-color);
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 23px 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:hover {
+  font-weight: 500;
+}
+
+.is-active,
+.nav__link:hover {
+  color: var(--blue-accent-color);
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active:after {
+  position: absolute;
+  right: 0;
+  left: 0;
+  bottom: 0;
+
+  content: "";
+  border-radius: 8px;
+  display: block;
+  background: var(--blue-accent-color);
+  height: 4px;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+  width: 1024px;
+  padding: 50px 40px;
+  margin: 0 auto;
+}
+
+.card,
+.cards {
+  justify-content: space-between;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  background: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 0.3s;
+  margin-bottom: 46px;
```

Comment:
follow BEM convention. blocks shouldn't have margins. instead set margins for .cards__card element